packages/pricing/src/migrations/Migration20230929122253.ts

```diff
@@ -101,5 +101,39 @@ export class Migration20230929122253 extends Migration {
     this.addSql(
       'alter table "price_rule" add constraint "price_rule_price_set_money_amount_id_foreign" foreign key ("price_set_money_amount_id") references "price_set_money_amount" ("id") on update cascade on delete cascade;'
     )
+
+    this.addSql(
+      'create table if not exists "price_list" ("id" text not null, "status" text check ("status" in (\'active\', \'draft\')) not null default \'draft\', "starts_at" timestamptz null, "ends_at" timestamptz null, "number_rules" integer not null default 0, constraint "price_list_pkey" primary key ("id"));'
+    )
+
+    this.addSql(
+      'create table "price_list_rule" ("id" text not null, "rule_type_id" text not null, "value" text not null, "priority" integer not null default 0, "price_list_id" text not null, constraint "price_list_rule_pkey" primary key ("id"));'
+    )
+
+    this.addSql(
+      'create index "IDX_price_list_rule_rule_type_id" on "price_list_rule" ("rule_type_id");'
+    )
+    this.addSql(
+      'create index "IDX_price_list_rule_price_list_id" on "price_list_rule" ("price_list_id");'
+    )
```

Comment:
May not be necessary since we are using `id` as the PK. But do we want to add an index for `(rule_type_id, price_list_id)` unique combination?